client/my-sites/upgrades/checkout-thank-you/business-plan-details.jsx

```diff
@@ -12,26 +12,28 @@ import i18n from 'lib/mixins/i18n';
 import { isBusiness } from 'lib/products-values';
 import PurchaseDetail from 'components/purchase-detail';
 
-const BusinessPlanDetails = ( { selectedSite, sitePlans } ) => {
+const BusinessPlanDetails = ( { selectedSite, sitePlans, selectedFeature } ) => {
 	const plan = find( sitePlans.data, isBusiness );
 
 	return (
 		<div>
-			{ plan.hasDomainCredit && <CustomDomainPurchaseDetail selectedSite={ selectedSite } /> }
-
-			<PurchaseDetail
-				icon="customize"
-				title={ i18n.translate( 'Find a new theme' ) }
-				description={ i18n.translate( 'All our premium themes are now available at no extra cost. Try them out now.' ) }
-				buttonText={ i18n.translate( 'Browse premium themes' ) }
-				href={ '/design/' + selectedSite.slug } />
+			{ ! selectedFeature &&
+				<PurchaseDetail
+					icon="customize"
+					title={ i18n.translate( 'Find a new theme' ) }
+					description={ i18n.translate( 'All our premium themes are now available at no extra cost. Try them out now.' ) }
+					buttonText={ i18n.translate( 'Browse premium themes' ) }
+					href={ '/design/' + selectedSite.slug } />
+			}
 
 			<PurchaseDetail
 				icon="stats-alt"
 				title={ i18n.translate( 'Stats from Google Analytics' ) }
 				description={ i18n.translate( 'Connect to Google Analytics for the perfect complement to WordPress.com stats.' ) }
 				buttonText={ i18n.translate( 'Connect Google Analytics' ) }
 				href={ '/settings/analytics/' + selectedSite.slug } />
+
+			{ plan.hasDomainCredit && <CustomDomainPurchaseDetail selectedSite={ selectedSite } /> }
```

Comment:
We wanted to bump the context-relevant card to the top, which is why this is at the bottom now.